open_rarity/resolver/opensea_api_helpers.py

```diff
@@ -131,23 +144,79 @@ def opensea_traits_to_token_metadata(asset_traits: list) -> TokenMetadata:
     )
 
 
-def get_collection_with_metadata(
+def get_tokens_from_opensea(
```

Comment:
naming wise we should use the same wording here as in method above. I'm fine with both `tokens` or `assets` in the naming.

Reply:
assets is from opensea, tokens is our object (`Token`) so it should be accurate